guides/common/modules/proc_updating-disconnected-server.adoc

```diff
@@ -6,6 +6,12 @@ Update your air-gapped {Project} setup where the connected {ProjectServer}, whic
 .Prerequisite
 * Back up your {ProjectServer}.
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+* Install `reposync` that is required for the updating procedure.
++
+[options="nowrap" subs="attributes"]
+----
+{package-install} yum-utils 'dnf-command(reposync)'
```

Comment:
On EL 8.8, running `dnf install yum-utils 'dnf-command(reposync)'` and `dnf install yum-utils` results in the same 8 packages that will be installed:

````
yum-utils
dbus-glib
dnf-plugins-core
python3-dateutil
python3-dbus
python3-dnf-plugins-core
python3-six
python3-systemd
````

Reply:
Right. I guess `yum-utils` does the job of `dnf-commmand(reposync)` by installing `dnf-plugins-core` along side `yum-utils`. Can we remove either of them? @ekohl

Reply:
There's no need to install `yum-utils` if you use `dnf repoclosure`
```suggestion
{package-install} 'dnf-command(reposync)'
```

Reply:
This was marked as resolved, but not actually done so I've submitted the suggestion as https://github.com/theforeman/foreman-documentation/pull/2423